Sources/PrimerSDK/Classes/Error Handler/PrimerValidationError.swift

```diff
@@ -107,6 +110,8 @@ public enum PrimerValidationError: PrimerErrorProtocol {
             return "invalid-phone-number-country-code"
         case .invalidOTPCode:
             return "invalid-otp-code"
+        case .invalidCardNetwork:
+            return "unsupported-card-network"
```

Comment:
we have changed this one to `unsupported-card-type` in order to be able to reuse this in future